app/src/main.cpp

```diff
@@ -348,121 +388,120 @@ namespace scitt
         .install();
 
       static constexpr auto get_entries_tx_ids_path = "/entries/txIds";
-      auto get_entries_tx_ids = [this](
-                                  EndpointContext& ctx,
-                                  nlohmann::json&& params) {
-        const auto parsed_query =
-          http::parse_query(ctx.rpc_ctx->get_request_query());
+      auto get_entries_tx_ids =
+        [this](EndpointContext& ctx, nlohmann::json&& params) {
+          const auto parsed_query =
+            http::parse_query(ctx.rpc_ctx->get_request_query());
 
-        std::string error_reason;
+          ccf::SeqNo from_seqno =
+            get_query_value<uint64_t>(parsed_query, "from").value_or(1);
+          std::optional<ccf::SeqNo> to_seqno_opt =
+            get_query_value<uint64_t>(parsed_query, "to");
+          ccf::SeqNo to_seqno;
 
-        size_t from_seqno;
-        if (!http::get_query_value(
-              parsed_query, "from", from_seqno, error_reason))
-        {
-          from_seqno = 1;
-        }
-
-        size_t to_seqno;
-        if (!http::get_query_value(parsed_query, "to", to_seqno, error_reason))
-        {
-          ccf::View view;
-          ccf::SeqNo seqno;
-          const auto result = get_last_committed_txid_v1(view, seqno);
-          if (result != ccf::ApiResult::OK)
+          if (to_seqno_opt.has_value())
           {
-            throw InternalError(fmt::format(
-              "Failed to get last committed transaction ID: {}",
-              ccf::api_result_to_str(result)));
+            to_seqno = *to_seqno_opt;
+          }
+          else
+          {
+            ccf::View view;
+            ccf::SeqNo seqno;
+            const auto result = get_last_committed_txid_v1(view, seqno);
+            if (result != ccf::ApiResult::OK)
+            {
+              throw InternalError(fmt::format(
+                "Failed to get last committed transaction ID: {}",
+                ccf::api_result_to_str(result)));
+            }
+            to_seqno = seqno;
           }
-          to_seqno = seqno;
-        }
 
-        if (to_seqno < from_seqno)
-        {
-          throw BadRequestError(
-            errors::InvalidInput,
-            fmt::format(
-              "Invalid range: Starts at {} but ends at {}",
-              from_seqno,
-              to_seqno));
-        }
+          if (to_seqno < from_seqno)
```

Comment:
clang-format works in mysterious ways, and somehow decided it was time to re-ident everything in this lambda :(